articles/dac-calculator-explainer/components/donut.js

```diff
@@ -1,150 +1,42 @@
-import { useEffect, useState, useRef } from 'react'
-import { Vega } from 'react-vega'
-import { useThemeUI, Box } from 'theme-ui'
-var vegaLite = require('vega-lite')
-
-const Donut = ({ results, initWidth, innerRadius }) => {
-  const [spec, setSpec] = useState(null)
-  const [loaded, setLoaded] = useState(false)
-  const [width, setWidth] = useState(null)
-  const container = useRef(null)
-  const { theme } = useThemeUI()
-  const { rawColors: colors } = theme
-
-  const updateWidth = (node) => {
-    if (container.current) {
-      const newWidth = container.current.offsetWidth * 0.85
-      if (newWidth < initWidth) {
-        setWidth(newWidth)
-      } else {
-        setWidth(initWidth)
-      }
-    }
-  }
-
-  useEffect(() => {
-    updateWidth(container)
-  }, [container.current])
-
-  useEffect(() => {
-    let id = null
-    const listener = () => {
-      clearTimeout(id)
-      id = setTimeout(() => {
-        updateWidth(container)
-      }, 150)
-    }
-    window.addEventListener('resize', listener)
-    return () => {
-      window.removeEventListener('resize', listener)
-    }
-  }, [])
-
-  useEffect(() => {
-    const config = {
-      background: null,
-      cursor: 'pointer',
-      view: {
-        stroke: null,
-      },
-    }
-
-    const spec = {
-      data: {
-        name: 'values',
-      },
-      mark: {
-        type: 'arc',
-        innerRadius: width * 0.2333,
-      },
-      encoding: {
-        theta: {
-          field: 'fraction',
-          type: 'quantitative',
-          scale: { domain: [0, 1] },
-        },
-        opacity: {
-          field: 'index',
-          type: 'quantitative',
-          scale: { domain: [0, 3], range: [0.3, 0.9] },
-          legend: null,
-        },
-        color: {
-          field: 'color',
-          type: 'quantitative',
-          scale: {
-            domain: [0, 1],
-            range: [colors.secondary, colors.purple],
-          },
-          legend: null,
-        },
-      },
-    }
-
-    setSpec(vegaLite.compile(spec, { config: config }).spec)
-    setLoaded(true)
-  }, [theme, width])
+import { Box } from 'theme-ui'
+import { Chart, Plot, Donut as DonutComponent } from '@carbonplan/charts'
 
+const Donut = ({ results, maxWidth, center = false }) => {
```

Comment:
small naming nit, for chart use cases like this i'd usually prefer to call the wrapper `DonutChart` rather than alias the component than comes from the library, and instead leave that with it's original name